lib/lorekeeper/fast_logger.rb

```diff
@@ -21,8 +21,29 @@ def initialize(file)
       @file = file # We only keep this so we can inspect where we are sending the logs
     end
 
-    LOGGING_METHODS = [:debug, :info, :warn, :error, :fatal]
-    METHOD_SEVERITY_MAP = { debug: DEBUG, info: INFO, warn: WARN, error: ERROR, fatal: FATAL }
+    LOGGING_METHODS = [
```

Comment:
It seems we also have a `TRACE` level. Maybe we could add it as `-1` . Kind of hacky but would allow the rest of the code to stay the same.

Reply:
thought about it but it's outside the scope of this PR that is growing large already. Let's have another story, me thinks.